.env-dev.template

```diff
@@ -38,6 +43,12 @@ PUBLIC_URL_PREFIX= # Set Tunnel URL if using local storage
 # AZURE_STORAGE_ACCOUNT_KEY=
 # AZURE_STORAGE_CONTAINER=
 
+# Create the S3 bucket
```

Comment:
Can u please change this sentence to "Set S3 Bucket name when using AWS speech services" in both .env.template and .env-dev.template files? Also you can move this S3_BUCKET_NAME to below AWS Speech/Translation Keys as they are only used there and nowhere else.